test/e2e/mv3-load-times.spec.js

```diff
@@ -0,0 +1,131 @@
+          true,
+        );
+      },
+    );
+  });
+  it('should measure the time for confirming a transaction', async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('[data-testid="eth-overview-send"]');
+
+        await driver.fill(
+          'input[placeholder="Search, public address (0x), or ENS"]',
+          '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+        );
+
+        const inputAmount = await driver.findElement('.unit-input__input');
+        await inputAmount.fill('1000');
+
+        const errorAmount = await driver.findElement('.send-v2__error-amount');
+        assert.equal(
+          await errorAmount.getText(),
+          'Insufficient funds.',
+          'send screen should render an insufficient fund error message',
+        );
+
+        await inputAmount.press(driver.Key.BACK_SPACE);
+        await inputAmount.press(driver.Key.BACK_SPACE);
+        await inputAmount.press(driver.Key.BACK_SPACE);
+        await driver.wait(async () => {
+          const sendDialogMsgs = await driver.findElements(
+            '.send-v2__form div.dialog',
+          );
+          return sendDialogMsgs.length === 1;
+        }, 10000);
+
+        await driver.assertElementNotPresent('.send-v2__error-amount');
+
+        const amountMax = await driver.findClickableElement(
+          '.send-v2__amount-max',
+        );
+        await amountMax.click();
+
+        let inputValue = await inputAmount.getProperty('value');
+
+        assert(Number(inputValue) > 24);
+
+        await amountMax.click();
+
+        assert.equal(await inputAmount.isEnabled(), true);
+
+        await inputAmount.fill('1');
+
+        inputValue = await inputAmount.getProperty('value');
+        assert.equal(inputValue, '1');
+
+        await driver.clickElement({ text: 'Next', tag: 'button' });
+
+        const timestampBeforeAction = new Date();
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        await driver.clickElement('[data-testid="home__activity-tab"]');
+        await driver.wait(async () => {
+          const confirmedTxes = await driver.findElements(
+            '.transaction-list__completed-transactions .transaction-list-item',
+          );
+          return confirmedTxes.length === 1;
+        }, 10000);
+
+        await driver.waitForSelector('.transaction-status--confirmed');
+        const timestampAfterAction = new Date();
+        loadingTimes.sendTx = timestampAfterAction - timestampBeforeAction;
+
+        assert.equal(
+          Object.prototype.hasOwnProperty.call(loadingTimes, 'sendTx'),
+          true,
+        );
```

Comment:
Since we are only measuring response time here, we can avoid assert.

Reply:
make sense. I've removed both then @jpuri